data/human/transport missions.txt

```diff
@@ -3304,3 +3304,90 @@ mission "Spacediving amateurs"
 	on complete
 		payment 30000
 		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."
+
+
+
+mission "Human to Hai Space - No Contact"
+	minor
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "tourism" "urban"
+	to offer
+		random < 2
+		not "First Contact: Hai: offered"
+	on offer
+		conversation
+			`	A young man carrying luggage approaches you. "You look like a knowledgeable captain," he says. "In all your travels, have you ever heard tell of any... intelligent rodents?"`
+			choice
+				`	"What are you talking about?"`
+					goto confused
+				`	"What, like in a zoo?"`
+					goto confused
+				`	"Are you high?"`
+			`	"No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
```

Comment:
since "Hai" and "high" are honophones, I think it would be funnier and less spoilery if you changed it around to use the other word:

```suggestion
			`	"No, I want to go to the high-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
```